l10n_it_account/tests/test_l10n_it_account.py

```diff
@@ -4,6 +4,16 @@
 
 class TestAccount(TransactionCase):
 
+    def _company_create(self):
+        self.company = self.env['res.company'].create({
+            'name': 'Italy test company'
+        })
+        country_id_it = self.env['res.country'].search([
+            ('code', '=', 'IT')
+        ], limit=1)
+        self.company.partner_id.country_id = country_id_it.id
+        return True
```

Comment:
```suggestion
        return self.env["res.company"].create({
            "name": "Italy test company",
            "country_id": self.env.ref("base.it").id,
        })
```
and save the variable at the call line.